spec/factories/organizations.rb

```diff
@@ -13,7 +13,11 @@
     end
 
     after(:create) do |organization, evaluator|
-      organization.webhook_endpoints.create(webhook_url: evaluator.webhook_url)
+      if evaluator.webhook_url
+        organization.webhook_endpoints.create!(webhook_url: evaluator.webhook_url)
+      end
+
+      organization.api_keys.create!
```

Comment:
could you instead of creating the api_keys here add it to the organizations factory?